common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -249,6 +251,7 @@ def student_view(self, context):
             'ajax_url': self.system.ajax_url,
             'next_url': context.get('next_url'),
             'prev_url': context.get('prev_url'),
+            'masquerade_override': masquerading and view_html is not None,
```

Comment:
The information we're passing is a boolean, `True` if we're overriding the "hide exam" behavior due to masquerading, `False` otherwise. It's used to trigger the `This exam is hidden from the learner` banner.